robot_sdk/wrappers/sim_wrapper.py

```
@@ -3,61 +3,40 @@
 """
 Python wrapper for Orangewood simulation SDK APIs
 """
-import time
+import getpass
 import sys
-import signal
-import rospy
 
-from owl_robot_sdk import OwlSimClient
+import rclpy
+from robogpt_tools.robot_sdk.SDK.owl_sim_sdk.robots import owl
```

Ruff (0.12.2)

`robogpt_tools.robot_sdk.SDK.owl_sim_sdk.robots.owl` imported but unused

Remove unused import: `robogpt_tools.robot_sdk.SDK.owl_sim_sdk.robots.owl`

(F401)